blocks/identity-block/components/FormInputField/index.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
This test file is not being picked up in the coverage checker - you can see that it's not listed in the table output at the bottom of the "Run tests with coverage" output https://github.com/WPMedia/fusion-news-theme-blocks/runs/3365681291?check_suite_focus=true

Reply:
I'm not sure why... and would like to know.  I don't suppose you have any ideas?

Reply:
I fixed the issue on canary, the folder wasn't part of the jest config